src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.sfdc.util;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * CommonActions.java
+ * Class with common methods that many classes can use.
+ */
+public final class CommonActions {
+
+    private static WebDriverWait waiter;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "waiter" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)